lib/infosections.py

```diff
@@ -0,0 +1,78 @@
+
+from lib.network import get_ip, get_tor_address
+from lib.qr_generator import generate_qr_code
+from lib.lnd import LndGRPC
+from lib.btc import BtcRPC
+import lib.rpc_pb2 as ln
+import lib.rpc_pb2_grpc as lnrpc
+from lib.network import get_tor_address
+from consts import bold_font
+
+import pygame
+
+class InfoSectionsList():
+
+    def __init__(self):
+        # Init RPC connections
+        self.btc_grpc = BtcRPC()
```

Comment:
These are both already instantiated from inside of UmbUI() so we could pass them through rather than re-instantiating

Reply:
They're not in your version for some reason